lib/scarpe/edit_box.rb

```diff
@@ -5,7 +5,7 @@ class EditBox < Scarpe::Widget
     display_properties :text, :height, :width
 
     def initialize(text = nil, height: nil, width: nil, &block)
-      @text = text || block.call
+      @text = text.nil? ? "" : text
```

Comment:
You can probably write this as `text || ""`. Technically there's a difference -- your way would leave a text value of false as false. I don't think we'd do anything useful with a text value of false, though.